### PR TITLE
feat(CX-3642): create artist preview bottom sheet modal

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-3642] 

### Description

| 1+ artworks by the artist | 0 artworks by the artist | Placeholder (it actually has the same height as the modal with the data, I just can't take a screenshot quick enough -_-') |
| --- | --- | --- |
|  |  |  |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- create artist preview bottom sheet modal (MyCollection) -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3642]: https://artsyproduct.atlassian.net/browse/CX-3642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dariakoko** [comment]
I will adjust the design when [this PR](https://github.com/artsy/eigen/pull/8654) is merged

**Comment by dariakoko** [comment]
I will adjust the design when https://github.com/artsy/eigen/pull/8654 is merged
- make the image bigger
- add artworks count

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          <Text color="red100" underline onPress={() => {}}>
+            Remove Artist
+          </Text>
+        </Join>
+      </Flex>
+    </BottomSheetView>
+  )
+}
+
+const ShareArtistCheckbox: React.FC<{ onCheckboxPress: () => void }> = ({ onCheckboxPress }) => {
```
Comment (lordkiz) [inline src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx:35]: If you intend to consume the value of the checkbox in another component, you probably want onCheckboxPress to callback with the value of the checkbox.

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  return (
+    <Touchable haptic onPress={onCheckboxPress}>
```
Comment (lordkiz) [inline src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx:37]:
```suggestion
    <Touchable haptic onPress={() => {const v = !isSelected; setIsSelected(v); onCheckboxPress(v)}}>
```

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModals.tsx [hunk 3]
```diff
@@ -43,6 +44,9 @@ export const MyCollectionBottomSheetModals: React.FC<{}> = () => {
         handleIndicatorStyle={{ backgroundColor: "black", width: 40, height: 4, borderRadius: 2 }}
       >
         {view === "Add" && <MyCollectionBottomSheetModalAdd />}
+        {
+          /* view === "Artist" && */ <MyCollectionBottomSheetModalArtistPreviewQueryRenderer artistID="banksy" />
```
Comment (lordkiz) [inline src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModals.tsx:48]: 👀

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx:35]: I think we can think of this later. Currently, we don't have the backend support for this

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModals.tsx (hunk 3 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModals.tsx:48]: The store is implemented in another ticket 👀

**Review by lordkiz — APPROVED** [review]
Since the raised issues will be addressed separately, feel free to rebase+merge @dariakoko 
Great work 👍

File: src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    `,
+  }
+)
+
+export const MyCollectionBottomSheetModalArtistPreviewQueryRenderer: React.FC<{
+  artistID: string
+  uploadsCount: number | null
+}> = ({ artistID, uploadsCount }) => {
+  return (
+    <QueryRenderer<MyCollectionBottomSheetModalArtistPreviewQuery>
```
Comment (MounirDhahri) [inline src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPrevie.tsx:109]: fwiw, I would very much recommend again using QueryRenderers anymore and instead use hooks

**Review by MounirDhahri — COMMENTED** [review]
Nice PR 👏

**Pull request merged**